go-controller/pkg/ovn/endpoints_test.go

```diff
@@ -33,58 +33,37 @@ func newEndpoints(name, namespace string, addresses []v1.EndpointAddress, ports
 	return &v1.Endpoints{
 		ObjectMeta: newEndpointsMeta(name, namespace),
 		Subsets: []v1.EndpointSubset{
-			v1.EndpointSubset{
+			{
 				Addresses: addresses,
 				Ports:     ports,
 			},
 		},
 	}
 }
 
-func (e endpoints) baseCmds(fexec *ovntest.FakeExec, isTCP bool) {
-	if isTCP {
-		fexec.AddFakeCmd(&ovntest.ExpectedCmd{
-			Cmd:    "ovn-nbctl --timeout=15 --data=bare --no-heading --columns=_uuid find load_balancer external_ids:k8s-cluster-lb-tcp=yes",
-			Output: K8S_TCP_LOAD_BALANCER_IP,
-		})
-	} else {
-		fexec.AddFakeCmd(&ovntest.ExpectedCmd{
-			Cmd:    "ovn-nbctl --timeout=15 --data=bare --no-heading --columns=_uuid find load_balancer external_ids:k8s-cluster-lb-udp=yes",
-			Output: K8S_UDP_LOAD_BALANCER_IP,
-		})
-	}
-}
+func (e endpoints) baseCmds(fexec *ovntest.FakeExec) {
```

Comment:
@alexanderConstantinescu baseCmds() is now an empty function, can it be removed?